mandatory/2-function-creation.js

```diff
@@ -6,7 +6,9 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  return typeof num === "number" && num % 2 === 0 && num <= 100;
```

Comment:
Brilliant.
Feel free to use brackets `()` if you think they make your code more readable:
`(typeof num === "number") && (num % 2 === 0) && (num <= 100);`
But totally optional